docs/tutorials/01_basics/05_control_flow.md

```diff
@@ -0,0 +1,68 @@
+# Control Flow
+
+## Primitive Types
+
+### Nil
+
+The most simplistic type is `Nil`. It has only a single value: `nil` and represents
+the absence of an actual value. If you ask which car occupies a parking spot, and
+no car is parking there, the answer is `nil`.
+
+### Bool
+
+The `Bool` type has just two possible values: `true` and `false` which represent the
+truth values of logic and Boolean algebra.
+
+```{.crystal, .crystal-play}
+p! true, false
+```
+
+[Boolean values](https://en.wikipedia.org/wiki/Boolean_data_type) are particularly useful for
+managing control flow in a program.
+
+## Boolean Algebra
+
+The following example shows operators for implementing [boolean algebra](https://en.wikipedia.org/wiki/Boolean_algebra) with
+boolean values:
+
+```{.crystal, .crystal-play}
+a = true
+b = false
+
+p! a && b, # AND
+   a || b, # OR
+   !a,     # NOT
+   a != b, # XOR
```

Comment:
Technically `^` is the XOR operator, though that's a bitwise operator.  `!=` works, but that's usually interpreted as "not equals", which, while accurate, seems strange.  If people search the docs for XOR, they're going to find `^`

Reply:
Yeah, I didn't want to bring bitwise operators into the picture at this point, because it's another concept and not relevant for boolean respective truthy/falsey. Not sure what the best thing to do here. Maybe keep `!=` and comment it as `inequality (=XOR)`?

Reply:
Yeah that seems like a nice way to put it.